### PR TITLE
A-BOHB example2

### PR DESCRIPTION
**Pull request by aaronkl**
new PR to replace #673 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-687-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-687/1/index.html

File: README.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ AutoGluon also provides state-of-the-art tools for neural hyperparameter and arc
 
 - [General introduction into HNAS](https://www.youtube.com/watch?v=pB1LmZWK_N8&feature=youtu.be)
 - [Introduction into HNAS with AutoGluon](https://www.youtube.com/watch?v=GJVwUyVWZas)
+- [Examples](https://github.com/awslabs/autogluon/tree/master/examples/nas/hnas)
```
Comment (daveharmon) [inline README.md:77]: if you're trying to link to the directory of the file you created here, this should be `https://github.com/awslabs/autogluon/tree/master/examples/hnas` right?

Comment (aaronkl) [reply README.md]: good catch! thanks

**Comment by szha** [comment]
Job PR-687-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-687/2/index.html

**Review by mseeger — APPROVED** [review]

**Comment by szha** [comment]
Job PR-687-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-687/3/index.html

**Review by Innixma — CHANGES_REQUESTED** [review]
Note: Plan to merge after modularization is complete

**Comment by szha** [comment]
Job PR-687-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-687/4/index.html

**Comment by Innixma** [comment]
@aaronkl Modularization has been completed. Could you rebase to mainline and edit the example to work with mainline's code?

**Comment by aaronkl** [comment]
@Innixma ok it's rebased now to current master branch

**Comment by szha** [comment]
Job PR-687-5 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-687/5/index.html

**Comment by aaronkl** [comment]
replaced by #765

**Pull request closed without merging**